languageserver/src/main/scala/langserver/core/LanguageServer.scala

```diff
@@ -13,7 +13,7 @@ import play.api.libs.json.JsObject
  * A language server implementation. Users should subclass this class and implement specific behavior.
  */
 class LanguageServer(inStream: InputStream, outStream: OutputStream) extends LazyLogging {
-  val connection = (new Connection(inStream, outStream)) { (method, params) =>
+  val connection: Connection = (new Connection(inStream, outStream)) { (method, params) =>
```

Comment:
it's been bugging me that `connection.showMessage` is been red in IJ because this was missing, guess I should consider switching editors soon ^^